tests/lib/reset.sh

```diff
@@ -201,9 +201,9 @@ reset_all_snap() {
 # snapd 2.49.2 installed
 #
 
-for snListLine in $(snap list --all | grep disabled); do
-    name="$(echo "$snListLine" | awk '{print $1}')"
-    rev="$(echo "$snListLine" | awk '{print $3}')"
+snap list --all | grep disabled | while read -r line ; do
```

Comment:
```suggestion
snap list --all | grep disabled | while read -r name version revision restOfLine ; do
```

And you can get rid of the awk invocations.

Reply:
looks good thanks for the suggestion!